examples/bri-3/src/bri/zeroKnowledgeProof/capabilities/createCCSMAnchorHash/createCCSMAnchorHashCommand.handler.ts

```diff
@@ -0,0 +1,27 @@
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { CCSMAnchorHashAgent } from '../../agents/ccsmAnchorHash.agent';
+import { CCSMAnchorHashStorageAgent } from '../../agents/ccsmAnchorHashStorage.agent';
+import { CreateCCSMAnchorHashCommand } from './createCCSMAnchorHash.command';
+
+@CommandHandler(CreateCCSMAnchorHashCommand)
+export class CreateCCSMAnchorHashCommandHandler
+  implements ICommandHandler<CreateCCSMAnchorHashCommand>
```

Comment:
interface?

Reply:
ICommandHandler is the command handler interface provided by nestjs for implementing the cqrs pattern. The corresponding command is handled by the "execute" method of this class.

Reply:
I thought we are not using interfaces anymore